pkg/blobaccess/wget/access.go

```diff
@@ -0,0 +1,128 @@
+package wget
+
+import (
+	gocontext "context"
+	"crypto/tls"
+	"io"
+	"net/http"
+
+	"github.com/open-component-model/ocm/pkg/blobaccess"
+	"github.com/open-component-model/ocm/pkg/blobaccess/bpi"
+	"github.com/open-component-model/ocm/pkg/contexts/credentials"
+	"github.com/open-component-model/ocm/pkg/contexts/credentials/builtin/wget/identity"
+	"github.com/open-component-model/ocm/pkg/contexts/ocm/cpi"
+	"github.com/open-component-model/ocm/pkg/errors"
+	"github.com/open-component-model/ocm/pkg/mime"
+	"github.com/open-component-model/ocm/pkg/optionutils"
+)
+
+const (
+	CACHE_CONTENT_THRESHOLD = 4096
+)
+
+func DataAccessForWget(url string, opts ...Option) (blobaccess.DataAccess, error) {
+	blobAccess, err := BlobAccessForWget(url, opts...)
+	if err != nil {
+		return nil, err
+	}
+	return blobAccess, nil
+}
+
+func BlobAccessForWget(url string, opts ...Option) (_ blobaccess.BlobAccess, rerr error) {
+	eff := optionutils.EvalOptions(opts...)
+	log := eff.Logger("URL", url)
+
+	creds, err := eff.GetCredentials(url)
+	if err != nil {
+		return nil, err
+	}
+	if creds == nil {
+		log.Debug("no credentials found for", "url", url)
+	}
```

Comment:
Does this mean, it's not possible to download something without credentials? Even when it's public available and not protected at all? (e.g. https://github.com/open-component-model/ocm/releases/latest)